submit_and_compare/submit_and_compare.py

```diff
@@ -147,8 +196,20 @@ def student_submit(self, submissions, suffix=''):
         '''
         Save student answer
         '''
+        if self.max_attempts > 0 and self.count_attempts >= self.max_attempts:
+            raise StandardError(
```

Comment:
Do we actually want to raise an error or should we log it and handle it gracefully?